subprojects/core/src/main/java/org/gradle/internal/scripts/ScriptLanguages.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradle.internal.scripts;
+
+import org.gradle.scripts.ScriptLanguage;
+
+/**
+ * Registry of scripting languages.
```

Comment:
It bothers me a bit that "scripting language" is used in the prose but `ScriptLanguage` in the code.

I suggest renaming the two new interfaces to `ScriptingLanguage` and `ScriptingLanguages` to match the prose.